nibiru/assetlist.json

```diff
@@ -139,6 +139,39 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
       }
     },
+    {
+      "description": "AXV",
+      "extended_description": "AXV is the Astrovault token.",
+      "socials": {
+        "website": "https://astrovault.io/",
+        "twitter": "https://x.com/axvdex"
+      },
+      "denom_units": [
+        {
+          "denom": "tf/nibi1vetfuua65frvf6f458xgtjerf0ra7wwjykrdpuyn0jur5x07awxsfka0ga/axv",
+          "exponent": 0
+        },
+        {
+          "denom": "AXV",
+          "exponent": 6
+        }
+      ],
+      "base": "tf/nibi1vetfuua65frvf6f458xgtjerf0ra7wwjykrdpuyn0jur5x07awxsfka0ga/axv",
+      "name": "AXV",
+      "display": "AXV",
+      "symbol": "AXV",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/tf/nibi1vetfuua65frvf6f458xgtjerf0ra7wwjykrdpuyn0jur5x07awxsfka0ga/axv.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/tf/nibi1vetfuua65frvf6f458xgtjerf0ra7wwjykrdpuyn0jur5x07awxsfka0ga/axv.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/tf/nibi1vetfuua65frvf6f458xgtjerf0ra7wwjykrdpuyn0jur5x07awxsfka0ga/axv.png",
```

Comment:
Instead of uploading these duplicate images, we should instead just reference the original case of them using image_sync.

```
{
  "image_sync": {
    "chain_name": "neutron",
    "base_denom": "cw20:neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af"
  },
  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.png",
  "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.svg"
}
```